### PR TITLE
Sil split

### PR DESCRIPTION
**Pull request by madhas8**
- [ ] I have read the [CONTRIBUTION GUIDE][contributing]

- [ ] This is a bug

- [ ] This is a feature request

- [ ] This is a syntax/style fix

- [ ] This is an improvement to code quality

---
**Please describe your changes below:**
(Note: A clear description of changes is required to approve any pull requests)

_"Changed the implementation for verification because..."_

[contributing]: http://dellemc-symphony.readthedocs.io/en/latest/contributingtosymphony.html

### CODE REVIEW

File: hdp-capability-registry-client-starter/pom.xml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+	<groupId>com.dell.cpsd</groupId>
+	<artifactId>hdp-capability-registry-client-starter</artifactId>
+	<packaging>jar</packaging>
+	<name>hdp-capability-registry-client-starter</name>
+	<description>Spring Boot Starter for hdp-capability-registry-client</description>
+
+	<parent>
+		<groupId>com.dell.cpsd</groupId>
+		<artifactId>hdp-capability-registry-client-parent</artifactId>
+		<version>1.2.0-SNAPSHOT</version>
```
Comment (ganapn) [inline hdp-capability-registry-client-starter/pom.xml]: This is the initial version of the POM. Can we have it 1.0.0?

File: hdp-capability-registry-client/pom.xml [hunk 2]
```diff
@@ -0,0 +1,257 @@
...
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+	<!-- Copyright (c) 2017 Dell Inc. or its subsidiaries. All Rights Reserved. -->
+	
+    <parent>
+		<groupId>com.dell.cpsd</groupId>
+		<artifactId>hdp-capability-registry-client-parent</artifactId>
+		<version>1.2.0-SNAPSHOT</version>
```
Comment (ganapn) [inline hdp-capability-registry-client/pom.xml]: Same as above

File: hdp-capability-registry-client/src/test/resources/rabbitmq.properties [hunk 3]
```diff
@@ -0,0 +1,8 @@
+remote.dell.amqp.rabbitHostname=${VM_FQDN}
+remote.dell.amqp.rabbitPort=5672
+remote.dell.amqp.rabbitUsername=${rabbit_user}
+remote.dell.amqp.rabbitPassword=${rabbit_password}
+remote.dell.amqp.rabbitVirtualHost=/
+remote.dell.amqp.rabbitIsSslEnabled=false
+data.center=datacenter
+application.name=system-integration-starter-test
```
Comment (ganapn) [inline hdp-capability-registry-client/src/test/resources/rabbitmq.properties]: Incorrect app name

File: pom.xml [hunk 4]
```diff
@@ -1,331 +1,179 @@
...
+		<version>1.0.1</version>
+	</parent>
 
     <modelVersion>4.0.0</modelVersion>
-    <artifactId>hdp-capability-registry-client</artifactId>
-    <version>1.1.0-SNAPSHOT</version>
-    
+    <groupId>com.dell.cpsd</groupId>
+    <artifactId>hdp-capability-registry-client-parent</artifactId>
+    <version>1.2.0-SNAPSHOT</version>
```
Comment (ganapn) [inline pom.xml]: If 1.1.0 is not released, we shouldn't upgrade the snapshot version

File: hdp-capability-registry-client-starter/pom.xml (hunk 1 above)
Comment (kumars182) [reply hdp-capability-registry-client-starter/pom.xml]: Yes

**Pull request merged**